package.json

```diff
@@ -10,6 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@chakra-ui/react": "^3.2.3",
```

Comment:
@KushalXCoder can we uninstall this?

```
npm uninstall @chakra-ui/react
```

I dont think we will need any additional UI component for the scope of this project as of now.

Once you do that, repeat the same steps of `git add .` `git commit` ,etc